proxylib/testparsers/blockparser.go

```diff
@@ -74,11 +74,11 @@ func getBlock(data [][]byte) ([]byte, int, int, error) {
 
 				// Now 'block' contains everything before the ':', parse it as a decimal number
 				// indicating the length of the frame AFTER the ':'
-				len64, err := strconv.ParseUint(block.String(), 10, 64)
-				if err != nil {
+				if lenUint64, err := strconv.ParseUint(block.String(), 10, 64); err != nil {
 					return block.Bytes(), 0, 0, err
+				} else {
+					block_len = int(lenUint64)
```

Comment:
As a matter of style, the else block after an error return seems off, but I understand it is used here to limit the scope of the (temporary) return variables.